opencog/atoms/core/VariableList.cc

```diff
@@ -228,29 +228,29 @@ void VariableList::get_vartype(const Handle& htypelink)
 		Type vt = TypeNodeCast(vartype)->get_value();
 		if (vt != ATOM)  // Atom type is same as untyped.
 		{
-			std::set<Type> ts = {vt};
+			TypeSet ts = {vt};
 			_varlist._simple_typemap.insert({varname, ts});
 		}
 	}
 	else if (TYPE_INH_NODE == t)
 	{
 		Type vt = TypeNodeCast(vartype)->get_value();
-		std::set<Type> ts;
-		std::set<Type>::iterator it = ts.begin();
+		TypeSet ts;
+		TypeSet::iterator it = ts.begin();
 		classserver().getChildren(vt, std::inserter(ts, it));
```

Comment:
These changes make the code hard to read, hard to understand ... is there a positive reason for these changes?

Reply:
No reason other than the fact that `std::set<Type>` appeared often throughout the code. Since we know (and I assume love) `HandleSet`, I thought introducing `TypeSet` would come naturally.

I tend to feel that C++ code contains too much punctuations, so when I get a chance to remove some (without impairing, in my opinion, understandability) I tend to go for it.